### PR TITLE
[FSDE-15288] Bulk actions support for hierarchical data

### PR DESCRIPTION
**Pull request by janibolkvadze**

### CODE REVIEW

File: src/chi-vue/src/components/checkbox/Checkbox.tsx [hunk 1]
```diff
@@ -29,21 +30,15 @@ export default class Checkbox extends Vue {
...
-      checkbox.classList.add(CHECKBOX_CLASSES.INDETERMINATE);
     } else {
       checkbox.indeterminate = false;
-      checkbox.classList.remove(CHECKBOX_CLASSES.INDETERMINATE);
     }
   }
 
   mounted() {
     this._updateCheckboxState();
   }
```
Comment (SergioQCL) [inline src/chi-vue/src/components/checkbox/Checkbox.tsx:40]: Should this be on the created hook?

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-349/1/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://a44b60d70649eaa6b6dbc96cada638dd47fd0b8b-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-349-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Review by SergioQCL — APPROVED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-349/2/. :x:

**Review by evansito — APPROVED** [review]

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://ac24365a40a53b852366a1ab5f5835e8dfe7a745-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-349-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Review by josecarlosgonzalezv — APPROVED** [review]

**Comment by jllr** [comment]
Great improvements on bulk actions component @janibolkvadze!

**Pull request merged**